Casks/w/workspace-one-intelligent-hub.rb

```diff
@@ -9,8 +9,8 @@
   homepage "https://www.getwsone.com/"
 
   livecheck do
-    url :url
-    strategy :extract_plist
+    url "https://docs-be.omnissa.com/api/bundle/workspace-one-intelligent-hub-for-macos-release-notesVSaaS/page/WorkspaceONEIntelligenHubmacOS-ReleaseNotes.html"
+    regex(/Workspace\s+ONE\s+Intelligent\s+Hub\s+v?(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/Intelligent\s+Hub\s+v?(\d+(?:\.\d+)+)/i)
```

The text also includes an instance of "Intelligent Hub 24.04", so dropping the leading "Workspace ONE" may make this a little more flexible.